packages/docs/docs/advanced/canvas-operations.mdx

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
+slug: /canvas-operations
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Canvas Operations
+
+Because Motion Canvas is build ontop of the Browser's 2D Rendering Context, we
+can make use several canvas operations that are provided by the Browser.
```

Comment:
I'm not sure if this section should be called "Canvas Operations" to be honest.
Any method of `CanvasRenderingContext2D` could be called a canvas operation.
This guide is more about Filters and Effects.